.github/workflows/node.js.yml

```diff
@@ -27,6 +27,24 @@ jobs:
         node-version: ${{ matrix.node-version }}
         cache: 'npm'
     - run: npm ci
+    - run: npm run generate
+    - name: Archive generated files
+      uses: actions/upload-artifact@v4
```

Comment:
What is this for?

Reply:
`npm run generate` is now necessary to be part of the build. Archiving the generated files might be useful if the build later fails on the generated files, since they can then be viewed via GitHub.

Alternatively it lets you view the effect on generated files via a pr, for example: https://github.com/ribrdb/desynced-tools/actions/runs/8217131145/artifacts/1312038323

But it's completely optional, just figured it would be useful.